src/cst1201/CharacterCounter.java

```diff
@@ -15,39 +15,38 @@
 public class CharacterCounter {
 
     public static void main(String[] args) {
-        System.out.print("Enter a string: ");
-
-        Scanner scanner = new Scanner(System.in);
-
-        String inputString = scanner.nextLine();
-        System.out.println("Your string was: " + inputString);
-
-        System.out.print("Enter a character: ");
-        char inputCharacter = scanner.next().charAt(0);
-
-        System.out.println("Your character was: " + inputCharacter);
-
-        int i = 0; //the index of the current character in inputString.
-        int characterCount = 0; //the number of characters found.
-
-        //traverse each character in the string.
-        while (i < inputString.length()) {
-            //if the current character matches the input character.
-            if (inputString.charAt(i) == inputCharacter) {
-                characterCount++; //increment a count.
-
-                //TODO: Make the character matching case insensitive.
-                //For example, "City Tech" and the character 't' should print 2
-                //occurrences.
-                //For homework, due Wednesday evening.
-            }
-
-            //move to the next character.
-            i++;
-        }
+ System.out.print("Enter a string: ");
+ Scanner scanner = new Scanner(System.in);
+ String inputString = scanner.nextLine();
+ System.out.println("Your string was: " + inputString);
+ System.out.print("Enter a character: ");
+ char inputCharacter = scanner.next().charAt(0);
+ System.out.println("Your character was: " + inputCharacter);
+ int i = 0; //the index of the current character in inputString.
+ int characterCount = 0; //the number of characters found.
+
+//Change all the characters in the string to lowercase
+String toLowerCase = inputString.toLowerCase();

```

Comment:
This is probably a bad variable name. Something like `inputStringInLowerCase` would be better.